### ISSUE TITLE
Analysis server crash from Flutter

### ISSUE THREAD

**Issue by fuedgabriel** [dart-lang/sdk#39473]
Analyzer Feedback from IntelliJ

## Version information

- `IDEA AI-191.8026.42.35.5900203`
- `2.6.0-edge.1103600280676ea169a30d7a503e836671cdc553`
- `AI-191.8026.42.35.5900203, JRE 1.8.0_202-release-1483-b03x64 JetBrains s.r.o, OS Windows 10(amd64) v10.0 , screens 1600x900, 1366x768`

## Exception
```
Dart analysis server, SDK version 2.6.0-edge.1103600280676ea169a30d7a503e836671cdc553, server version 1.27.3, FATAL error: Failed to handle request: {id: 1706, method: edit.getFixes, params: {file: C:\Users\House\AndroidStudioProjects\flutter_netflix_ui_redesign-master\lib\widgets\menu.dart, offset: 1347}, clientRequestTime: 1574304193332}
type 'ListLiteralImpl' is not a subtype of type 'SimpleIdentifier' in type cast
#0      FixProcessor._addFix_importLibrary_withExtension (package:analysis_server/src/services/correction/fix_internal.dart:2805:34)
#1      _AsyncAwaitCompleter.start (dart:async-patch/async_patch.dart:43:6)
#2      FixProcessor._addFix_importLibrary_withExtension (package:analysis_server/src/services/correction/fix_internal.dart:2804:51)
#3      FixProcessor.compute (package:analysis_server/src/services/correction/fix_internal.dart:474:13)
#4      _asyncThenWrapperHelper.<anonymous closure> (dart:async-patch/async_patch.dart:71:64)
#5      _rootRunUnary (dart:async/zone.dart:1132:38)
#6      _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#7      _FutureListener.handleValue (dart:async/future_impl.dart:137:18)
#8      Future._propagateToListeners.handleValueCallback (dart:async/future_impl.dart:678:45)
#9      Future._propagateToListeners (dart:async/future_impl.dart:707:32)
#10     Future._completeWithValue (dart:async/future_impl.dart:522:5)
#11     Future._asyncComplete.<anonymous closure> (dart:async/future_impl.dart:552:7)
#12     _rootRun (dart:async/zone.dart:1124:13)
#13     _CustomZone.run (dart:async/zone.dart:1021:19)
#14     _CustomZone.runGuarded (dart:async/zone.dart:923:7)
#15     _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:963:23)
#16     _microtaskLoop (dart:async/schedule_microtask.dart:41:21)
#17     _startMicrotaskLoop (dart:async/schedule_microtask.dart:50:5)
#18     _runPendingImmediateCallback (dart:isolate-patch/isolate_patch.dart:116:13)
#19     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:173:5)
```

For additional log information, please append the contents of
file://C:\Users\House\AppData\Local\Temp\report.txt.

**Comment by srawlins** [comment]
Thanks for reporting this issue! This looks to be a duplicate of https://github.com/dart-lang/sdk/issues/38437, which should be fixed in Dart 2.6.0 final. Please upgrade to the latest Dart. If you encounter this issue again, please re-open!

**Comment by srawlins** [comment]
Duplicate of https://github.com/dart-lang/sdk/issues/38437
